packages/frontend/src/components/common/Settings/SettingsCard.tsx

```diff
@@ -3,7 +3,7 @@ import { FC } from 'react';
 
 const SettingsCard: FC<PaperProps> = ({ children, ...rest }) => {
     return (
-        <Paper shadow="sm" withBorder p="md" {...rest}>
+        <Paper shadow="sm" withBorder p="md" maw={1200} {...rest}>
```

Comment:
This won't fix it since this is controlled by our generic Page component in `packages/frontend/src/components/common/Page/Page.tsx`.

Our pages can have:
- a fixed width ( using `withFixedContent` ) 
- minimum width possible ( using `withFitContent` )
- use all the width ( no prop )


When applying a fixed with we default to 900px:
```
export const PAGE_CONTENT_WIDTH = 900;
```

Changing this value would impact all the pages with `withFixedContent`, which might be a bit percentage of the app.

@IrakliJani might know more about the impact of changing this default value.

Reply:
yep, I talked to him and he said he'd have to make this change himself :) 
I'm going to close this ticket...